### PR TITLE
docs/actions: Update URL of the codegen contrib guide

### PR DESCRIPTION
**Pull request by oliverpool**
### Description
The current [docs](https://hasura.io/docs/1.0/graphql/manual/actions/codegen.html#building-a-codegen-for-your-framework) links to https://github.com/hasura/codegen-builder-contrib/, which is archived.
This PR changes the URL to the location of https://github.com/hasura/codegen-assets/blob/master/builder-kit/README.md (since the root README.md does not contain much information about creating a custom code generator

### Changelog
-  `no-changelog-required`

### Affected components
- [x] Docs

#### Breaking changes

- [ ] No Breaking changes

_Edit: see it on Netlify here https://deploy-preview-5323--hasura-docs.netlify.app/graphql/manual/actions/codegen.html#building-a-codegen-for-your-framework_

### CODE REVIEW

**Comment by oliverpool** [comment]
I really like this project, but I am sorry, I can't sign your CLA.

https://drewdevault.com/2018/10/05/Dont-sign-a-CLA.html

**Comment by marionschleifer** [comment]
Hey @oliverpool, thank you for your contribution 🙂  Do you mind if we include this change in a bigger refactor PR?

**Comment by sezgi** [comment]
@oliverpool thanks for bringing up your concerns with the CLA. The purpose of the CLA is to make sure:

- GraphQL Engine and people who download GraphQL Engine have the right to use the contribution,
- The contributor is not violating anyone else's rights by making this contribution, e.g. by copying code from somewhere else.

File: docs/graphql/manual/actions/codegen.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ As of now, Hasura provides codegen for a few frameworks (``nodejs-express``,
 ``typescript-zeit``, etc).
 
 If you wish to build a code generator for your framework
-`read the contrib guide <https://github.com/hasura/codegen-builder-contrib/>`_.
+`read the contrib guide <https://github.com/hasura/codegen-assets/blob/master/builder-kit/README.md>`_.
```
Comment (sezgi) [inline docs/graphql/manual/actions/codegen.rst:117]: why not https://github.com/hasura/codegen-assets/tree/master/builder-kit ?

Comment (oliverpool) [reply docs/graphql/manual/actions/codegen.rst:117]: That would be fine as well (I don't think that seeing the other files of this directory helps at all)

**Comment by oliverpool** [comment]
@marionschleifer no, sure. Feel free to take it to another PR.

---

>  thanks for bringing up your concerns with the CLA. The purpose of the CLA is to make sure:

@sezgi thank you for your reply, let me detail my concerns:

> GraphQL Engine and people who download GraphQL Engine have the right to use the contribution

I totally agree that all contributions must agree to the license of the repo (Apache License, Version 2.0). But as far as I know, you don't need a CLA for this.

> The contributor is not violating anyone else's rights by making this contribution, e.g. by copying code from somewhere else.

I guess you are referring to a `developer certificate of origin`. Quote from https://drewdevault.com/2018/10/05/Dont-sign-a-CLA.html
> What about the Linux kernel developer certificate of origin? I applaud the Linux kernel’s approach here. It covers their bases while still strongly protecting the rights of the patch owner. It’s a short statement with little legalese and little fanfare to agreeing to it (just add “Signed-off By” to your commit message). I approve.

I also agree with this statement.
---

**However the 2nd point of the [CLA](https://cla-assistant.io/hasura/graphql-engine?pullRequest=5323) of this project is**:
> Grant of Copyright License. Subject to the terms of this Agreement, You hereby grant to Hasura and to recipients of software distributed by Hasura a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, and distribute your contributions and such derivative works.

I think this goes far beyond "GraphQL Engine and people who download GraphQL Engine have the right to use the contribution".
It means that you can use my work and distribute it under whichever license you want! (proprietary for instance...) - and I can't agree to that. 

PS: the actual change of this PR is trivial and is likely out of scope for any CLA whatsoever. My point is more general (could be moved to a separate issue if you prefer)

**Comment by sezgi** [comment]
@oliverpool thanks for your feedback; will discuss this with the team. In the meantime, we'll take this to a different PR as @marionschleifer suggested.

**Comment by oliverpool** [comment]
@sezgi did you have any time to bring this up with your team? (I guess the licensing issue is related to your enterprise offering with added features - I hope we can find a way to let you add proprietary addons while letting anyone contribute without requiring a CLA).

I am playing with Hasura along Cloudflare workers and Google Firebase, and it works really good!
I would really enjoy contributing to this project, once this minor legal issue is cleared

**Comment by oliverpool** [comment]
@coco98 so I guess that you are not willing to consider removing the need for a CLA, right?

**Pull request closed without merging**